content/references/rippled-api/transaction-formats/transaction-common-fields.md

```diff
@@ -66,11 +66,11 @@ The only flag that applies globally to all transactions is as follows:
 
 | Flag Name             | Hex Value  | Decimal Value | Description               |
 |:----------------------|:-----------|:--------------|:--------------------------|
-| `tfFullyCanonicalSig` | `0x80000000` | 2147483648    | _(Strongly recommended)_ Require a fully-canonical signature. |
+| `tfFullyCanonicalSig` | `0x80000000` | 2147483648  | **DEPRECATED** No effect. Require a fully-canonical signature. |
```

Comment:
@mDuo13 , should this be "requires"?

Reply:
No, it's an imperative. As in, "The instructions to the server are, if this flag is enabled: require a fully canonical signature for this transaction."